scripts/generate_root_ca_certs

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+# Commit that updated the certificates for Android 10.
+readonly COMMIT="be85a54767b3559e7bb262e4f1a78e1c5829e478"
+
+readonly DOWNLOAD_DIR="${SCRIPTS_DIR}/../target/tmp"
```

Comment:
Why is this under `target`? That is the directory that cargo uses for its own artifacts. I would create a dedicated top level folder. In fact, it seems most of this logic is emulating something like git submodules, have you considered just using that? Or is the ca-certificates not a stand-alone git repo?

Reply:
Changed it to use a `downloads` directory and added it to `.gitignore`. I did consider using a git sub-module, but was not sure how to keep it pinned to a specific commit unless we explicitly decide to change it. As far as I understand, a 'git pull' will recursively pull new changes from sub-modules as well.